runtimes/eden/src/xcm_config.rs

```diff
@@ -97,8 +107,8 @@ pub type CurrencyTransactor = CurrencyAdapter<
 	(),
 >;
 parameter_types! {
-		// One XCM operation is 1_000_000_000 weight - almost certainly a conservative estimate.
-		pub UnitWeightCost: u64 = 1_000_000_000;
+		// One XCM operation is 200_000_000 weight - almost certainly a conservative estimate.
+		pub UnitWeightCost: u64 = 200_000_000;
```

Comment:
Is this a weight that should be benchmarked?

Reply:
yes